Epsilon.Host.Frontend/src/views/PerformanceDashboard.vue

```diff
@@ -35,20 +35,25 @@ import PersonalDevelopmentGraph from "@/components/PersonalDevelopmentGraph.vue"
 
 const data: Ref<CompetenceProfile | undefined> = ref(undefined)
 const App = new Api()
+const selectedTerm = ref<EnrollmentTerm | null>()
 
-const selectedTerm = ref<EnrollmentTerm | null>(null)
+function getTermDate(): string | null | undefined {
+    const index = data.value?.terms?.indexOf(selectedTerm.value) as number
+    const term = data.value?.terms?.at(index > 0 ? index - 1 : index)
+    return index > 0 ? term?.start_at : term?.end_at
```

Comment:
Maybe it is clearer to separate into one if statement of index > 0 with an else and then use that to include the return etc.